test/common/rhmi_config.go

```diff
@@ -0,0 +1,55 @@
+package common
+
+import (
+	"github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"testing"
+
+	goctx "context"
+)
+
+const (
+	RHMIConfigCRName  = "rhmi-config"
+	OperatorNamespace = "redhat-rhmi-operator"
+)
+
+// TestIntegreatlyRoutesExist tests that the routes for all the products are created
+func TestRHMIConfigCRs(t *testing.T, ctx *TestingContext) {
+	t.Log("Test rhmi config cr creation")
+
+	rhmiConfig := &v1alpha1.RHMIConfig{
+		TypeMeta: v1.TypeMeta{},
+		ObjectMeta: v1.ObjectMeta{
+			Name:      RHMIConfigCRName,
+			Namespace: OperatorNamespace,
```

Comment:
There's a `RHMIOperatorNamespace` if you want to use that instead to prevent duplication
https://github.com/integr8ly/integreatly-operator/blob/df1dcb34ab3656f994993c4808ba3cfa7f431e11/test/common/types.go#L18